cel/library.go

```diff
@@ -1746,3 +1750,12 @@ func (u *uri) take(char byte) bool {
 	}
 	return false
 }
+
+func (u *uri) takeDoubleSlash() bool {
+	first := u.take('/')
+	return first && u.take('/')
```

Comment:
Go [short-circuits binary expressions](https://go.dev/play/p/X4YSB1FsTaU):

```suggestion
	return u.take('/') && u.take('/')
```

Reply:
`u.take('/') && u.take('/')` is incorrectly flagged by [SA4000](https://staticcheck.dev/docs/checks/#SA4000).

Reply:
Ah go ahead and add a nolint directive. Should look like `//nolint:staticcheck // false positive of SA4000